### PR TITLE
Add Polish localization - Home page

### PR DESCRIPTION
**Pull request by mfilocha**
This is a part of the initial Polish localization - see: https://github.com/kubernetes/website/pull/18419

### CODE REVIEW

File: content/pl/docs/home/_index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+linkTitle: "Strona główna"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "Dokumentacja"
+    weight: 20
+    post: >
+      <p>Naucz się, jak korzystać z Kubernetes z pomocą dokumentacji, która opisuje pojęcia, zawiera samouczki i informacje źródłowe. Możesz także <a href="/editdocs/" data-auto-burger-exclude>pomóc w jej tworzeniu</a>!</p>
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
      <p>Naucz się, jak korzystać z Kubernetesa z pomocą dokumentacji, która opisuje pojęcia, zawiera samouczki i informacje źródłowe. Możesz także <a href="/editdocs/" data-auto-burger-exclude>pomóc w jej tworzeniu</a>!</p>
```

File: content/pl/docs/home/_index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+overview: >
+  Kubernetes to otwarte oprogramowanie służące do automatyzacji procesów uruchamiania, skalowania i zarządzania aplikacjami w kontenerach. Gospodarzem tego projektu o otwartym kodzie źródłowym jest Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>).
+cards:
+- name: concepts
+  title: "Podstawy"
+  description: "Poznaj podstawowe pojęcia Kubernetes."
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  description: "Poznaj podstawowe pojęcia projektu Kubernetes."
```

File: content/pl/docs/home/_index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  button: "Podstawowe pojęcia"
+  button_path: "/docs/concepts"
+- name: tutorials
+  title: "Wypróbuj Kubernetes"
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  title: "Wypróbuj Kubernetesa"
```

File: content/pl/docs/home/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  description: "Skorzystaj z samouczków, aby nauczyć się uruchamiać aplikacje na Kubernetes."
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  description: "Skorzystaj z samouczków, aby nauczyć się uruchamiać aplikacje na Kubernetesie."
```

File: content/pl/docs/home/_index.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  button: "Samouczki"
+  button_path: "/docs/tutorials"
+- name: setup
+  title: "Uruchom klaster"
+  description: "Uruchom klaster Kubernetes dopasowany do Twoich potrzeb i możliwości."
+  button: "Uruchom Kubernetes"
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  button: "Uruchom Kubernetesa"
```

File: content/pl/docs/home/_index.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  button_path: "/docs/setup"
+- name: tasks
+  title: "Naucz się korzystać z Kubernetes"
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  title: "Naucz się korzystać z Kubernetesa"
```

File: content/pl/docs/home/_index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  description: Zapoznaj się z terminologią, składnią poleceń, typami zasobów API i dokumentacją narzędzi instalacyjnych.
+  button: Zajrzyj do źródeł
+  button_path: /docs/reference
+- name: contribute
+  title: Weź udział w tworzeniu dokumentacji
+  description: Każdy może przyczynić się do tworzenia dokumentacji - zarówno nowicjusze, jak i starzy wyjadacze.
+  button: Weź udział
+  button_path: /docs/contribute
+- name: download
+  title: Pobierz Kubernetes
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  title: Pobierz Kubernetesa
```

File: content/pl/docs/home/_index.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  description: Jeśli instalujesz lub aktualizujesz Kubernetes, zajrzyj do informacji o najnowszym wydaniu.
```
Comment (kpucynski) [inline content/pl/docs/home/_index.md]:
```suggestion
  description: Jeśli instalujesz lub aktualizujesz Kubernetesa, zajrzyj do informacji o najnowszym wydaniu.
```

File: content/pl/docs/home/supported-doc-versions.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: Wspierane wersje dokumentacji Kubernetes
```
Comment (kpucynski) [inline content/pl/docs/home/supported-doc-versions.md]:
```suggestion
title: Wspierane wersje dokumentacji Kubernetesa
```

File: content/pl/docs/home/supported-doc-versions.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+content_template: templates/concept
+card:
+  name: about
+  weight: 10
+  title: Wspierane wersje dokumentacji
+---
+
+{{% capture overview %}}
+
+Ten serwis zawiera dokumentację do bieżącej i czterech poprzednich wersji Kubernetes.
```
Comment (kpucynski) [inline content/pl/docs/home/supported-doc-versions.md]:
```suggestion
Ten serwis zawiera dokumentację do bieżącej i czterech poprzednich wersji Kubernetesa.
```

File: content/pl/docs/home/_index.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
+---
+approvers:
+- chenopis
```
Comment (rlenferink) [inline content/pl/docs/home/_index.md]: Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having _a lot_ of people, files can be appointed to specific contributors.

Comment (mfilocha) [reply content/pl/docs/home/_index.md]: Fixed.

File: content/pl/docs/home/_index.md (hunk 2 above)
Comment (mfilocha) [reply content/pl/docs/home/_index.md]: I'v reworded the whole sentence.

**Comment by sftim** [comment]
/hold
pending approval & merge for #18419. Once that merges, it's OK to /unhold
People can /lgtm (for localization review) and /approve, even whilst this is held.

**Comment by sftim** [comment]
@nvtkaszpir: to mark this as reviewed and acceptable, comment with `/lgtm` on a line by itself

**Comment by rlenferink** [comment]
/unassign @daminisatya
(this will be taken care of by the Polish localization team once the initial PR merges)

**Comment by sftim** [comment]
Thanks @mfilocha & @nvtkaszpir
/approve
(Once the initial localization lands, there will be a set of approvers for Polish who can self-manage the localization)

**Comment by rlenferink** [comment]
_Approved based on @sftim being an [sig-docs approver](https://github.com/kubernetes/website/commit/756e5976846d7a112a833ffe7e448eb734ce17d6) (however, the `dev-1.17-pl.1` branch is a bit behind master)_

**Pull request merged**